cristin-project/src/main/java/no/unit/nva/cristin/projects/common/QueryBuilderProject.java

```diff
@@ -305,6 +326,18 @@ public QueryBuilderProject withParticipant(String participant) {
         return this;
     }
 
+    /**
+     * Setter a category of the project by name. [withDuplicates]
+     */
+    public QueryBuilderProject withCategory(String category) {
+        var categories =
+            query.containsKey(CATEGORY)
+                ? query.getValue(CATEGORY) + "," + category
```

Comment:
Har ikke testet dette, men getvalue(key) returnerer null hvis den ikke eksisterer, så det skulle være mulig å gjøre en `String.join(",",getvalue(key), value)`
Men koden du har skrevet er tydelig, ikke poeng i å forandre, med mindre noen andre har synspunkter på dette.

Reply:
i just copied your existing code...

Reply:
i think the result will be `null,value,value2,null` etc if not checking `containsKey` first